src/containers/EventCardGroup.jsx

```diff
@@ -32,6 +33,7 @@ const EventCardGroupContainer = ({ query }) => {
   const [searchInProgress, setSearchInProgress] = React.useState(false);
   const [allEvents, setAllEvents] = React.useState([]);
   const [visibleEvents, setVisibleEvents] = React.useState([]);
+  useDocumentTitle(`${searchQuery} - Search`);
```

Comment:
I would personally swap this.

```suggestion
  useDocumentTitle(`Search - ${searchQuery}`);
```